### DISCUSSION TITLE
I can´t open Miktex-Ubuntu

### DISCUSSION THREAD

**Discussion by alanro170694** [MiKTeX/miktex#1472] [Q&A] [answered]
Hi! 

I'm using Ubuntu 22.04.4 LTS and Miktex was working fine, buuuut a few days ago when I clicked in the icon the console didn't open. Only the "send error report" appear :( Can i open it in a different way? Or how can i uninstall it from the terminal? 

Cheers!

**Comment by tayllana** [comment]
It seems like you're having trouble opening MiKTeX on Ubuntu 22.04.4 LTS. If the application isn't launching properly and you're unable to open it through the usual means, you can try launching it from the terminal to see if it provides any error messages that could help diagnose the issue.

To open MiKTeX from the terminal, you can follow these steps:

1. Open a terminal window. You can do this by searching for "Terminal" in the Ubuntu application launcher, or by using the keyboard shortcut `Ctrl + Alt + T`.

2. Once the terminal is open, you can try running the command to launch MiKTeX. Typically, the command to launch MiKTeX from the terminal is `miktex-console`. You can try running this command by typing it into the terminal and pressing Enter.

   ```bash
   miktex-console
   ```

3. If MiKTeX encounters any errors while launching, it should display error messages in the terminal. These error messages can help diagnose the issue and troubleshoot the problem further.

If MiKTeX still doesn't launch properly or if you encounter any errors, you can try reinstalling it. To uninstall MiKTeX from the terminal, you can use the following command:

```bash
sudo apt-get remove miktex
```

This command will uninstall MiKTeX from your system. After uninstalling MiKTeX, you can reinstall it using the instructions provided by the MiKTeX website or any other method you prefer.

If you still encounter issues after reinstalling MiKTeX, you may need to seek further assistance, possibly from the MiKTeX community or support channels.

**Reply by alanro170694** [reply]
Thank you for  the information! 
I tried to run it from the terminal and this is the message i got:
Warning: Ignoring WAYLAND_DISPLAY on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

I think the best option is reinstalling the application hahaha

**Comment by rafaelcosta723** [comment, marked as answer]
Uninstalling and reinstalling again will not resolve the issue. Just install the missing packages, in this case run the following command `sudo apt-get install qt6-qpa-plugins
`

**Reply by alanro170694** [reply]
Thaaaaaaaaaaank you! It worked and now I can open the Miktek console 🙌🏾 What does qt6 mean? 😯

**Reply by waelnoby** [reply]
Thank you 100%

**Reply by Gholamrezadar** [reply]
This works. Thanks
